scarb/src/compiler/plugin/proc_macro/ffi.rs

```diff
@@ -110,7 +111,11 @@ impl ProcMacroInstance {
     pub fn declared_attributes_and_executables(&self) -> Vec<String> {
```

Comment:
Should we include no-op `declared_attributes` to `declared_attributes` as well, or is there a reason not to do so? 🤔

Reply:
We should not. The `declared_attributes_and_executables` declares attributes as allowed in the compiler, the `declared_attributes` is used by cairo-ls to obtain expandable attributes list.